src/data/markdown/docs/20 jslib/01 jslib/01 aws/00 S3Client.md

```diff
@@ -7,9 +7,11 @@ excerpt: 'S3Client class allows interacting with AWS S3 buckets and objects'
 
 <BlockingAwsBlockquote />
 
-S3Client allows interacting with AWS S3's buckets and objects. Namely, it allows the user to list buckets and the objects they contain, as well as download, uploading, and deleting objects. The S3Client operations are blocking, and we recommend reserving their usage to the [`setup`](/using-k6/test-lifecycle/) and [`teardown`](/using-k6/test-lifecycle/) functions as much as possible.
+`S3Client` interacts with the AWS Simple Storage Service (S3).
 
-S3Client is included in both the dedicated jslib `s3.js` bundle, and the `aws.js` one, containing all the services clients.
+With it, the user can list buckets, list objects in a bucket, download objects from a bucket, upload objects to a bucket, delete objects from a bucket, copy objects from one bucket to another, create multipart uploads, upload parts in a multipart upload, complete multipart uploads, and abort multipart uploads.
+
+Both the dedicated `s3.js` jslib bundle and the all-encompassing `aws.js` bundle include the `S3Client`.
```

Comment:
Nice editing here 👍 .

A small suggestion, and I'll leave it up to you, you could shorten the list of actions and say something like: 

"With it, you can do several operations such as list buckets, list objects in a bucket, or download objects from a bucket. For a full list of supported operations, see [Methods](#methods)."

That way we don't have to add to this paragraph if we add new methods here (although I don't know if this changes much?)

Reply:
You're absolutely right, I was actually looking for a way to make it more generic indeed (and not needing to much updates in the future either). I'll rephrase 👍🏻